composer.json

```diff
@@ -10,7 +10,7 @@
         }
     ],
     "require": {
-        "vimeo/psalm": "^3 || ^4 || ^5"
+        "vimeo/psalm": "^5"
```

Comment:
That means a breaking change, so I will bump the major version number. Probably time to release 1.x.